util/err.h

```diff
@@ -11,4 +11,8 @@ __attribute__((noreturn)) void verr(int eval, const char *fmt, va_list args) __a
 __attribute__((noreturn)) void verrx(int eval, const char *fmt, va_list args) __asm__("libutil_verrx");
 __attribute__((noreturn)) void err(int eval, const char *fmt, ...) __asm__("libutil_err");
 __attribute__((noreturn)) void errx(int eval, const char *fmt, ...) __asm__("libutil_errx");
+
+const char* libuitl_getprogname (void);
+void libuitl_setprogname(const char *);
+
```

Comment:
The man pages say these should be in stdlib.h. Do we have an existing wrapper for this?

Reply:
Not that I am aware of. Are you suggesting pulling these functions out into `stdlib.h/.c` files?

Reply:
they can probably be in the the existing cpp file, but we should get a stdlib.h wrapper like existing ones since that's where it's expected to be.

Reply:
I moved them to their own file to follow the pattern used by functions in other wrappers (`isatty.c` -> `unistd.h:libutil_isatty`). Created the wrapper, modified the Makefile, built it, and everything _seems_ ok.